lib/logger/logger.go

```diff
@@ -341,3 +399,36 @@ func SetOutputForTests(writer io.Writer) { output = writer }
 
 // ResetOutputForTest set logger output to default value
 func ResetOutputForTest() { output = os.Stderr }
+
+// TLSConfig creates tls.Config object from provided arguments
+func TLSConfig(certFile, keyFile, CAFile, serverName string, insecureSkipVerify bool) (*tls.Config, error) {
```

Comment:
had to add this function here, since import from httputils causes import cycle error.

Reply:
The only workaround would be to create a package for this function, but I would suggest to keep here for now.
We can move this into a single package later if needed